docker/script/bootstrap.sh

```diff
@@ -10,6 +10,7 @@ pip3 install wheel
 
 # On RHL and Centos based linux, openssl needs to be set as Python Curl SSL library
 export PYCURL_SSL_LIBRARY=openssl
+pip3 install --upgrade pip
```

Comment:
This change is already in `main`

Reply:
There are multiple commits to be merged, it is the other commit which contains a valuable big fix

Reply:
@o-nikolas do you want me to raise a PR for the single commit that contains the change?

It will not cause any issue to merge the above change into main even if it has been cherry picked already.